plugins/janus_audiobridge.c

```diff
@@ -2831,16 +2863,90 @@ void janus_audiobridge_incoming_rtp(janus_plugin_session *handle, int video, cha
 		int plen = 0;
 		const unsigned char *payload = (const unsigned char *)janus_rtp_payload(buf, len, &plen);
 		if(!payload) {
-			JANUS_LOG(LOG_ERR, "[Opus] Ops! got an error accessing the RTP payload\n");
+			JANUS_LOG(LOG_ERR, "(participant %"SCNu64")[Opus] Ops! got an error accessing the RTP payload\n", participant->user_id);
 			g_free(pkt->data);
 			g_free(pkt);
 		  	participant->working = FALSE;
 			return;
 		}
-		pkt->length = opus_decode(participant->decoder, payload, plen, (opus_int16 *)pkt->data, BUFFER_SAMPLES, USE_FEC);
+
+
+		//Check sequence number received, verify if relevant to expected one.
+		if(pkt->seq_number == participant->expected_seq) {
+			/* regular decode */
+            //output_samples = max_frame_size;
+            pkt->length = opus_decode(participant->decoder, payload, plen, (opus_int16 *)pkt->data, BUFFER_SAMPLES, 0);
+
+            //update last_timestamp
+            participant->last_timestamp = pkt->timestamp;
+
+            //Increment according to previous seq_number
```

Comment:
All the above lines have broken indentation (use tabs, not spaces).

Reply:
done